php/pantheon/checks/objectcache.php

```diff
@@ -29,11 +29,14 @@ public function run() {
 		} else {
 			$this->alerts[] = array("message"=> "object-cache.php exists", "code" => 0);
 		}
-
 		if ( ! defined( 'WP_REDIS_OBJECT_CACHE' ) || ! WP_REDIS_OBJECT_CACHE ) {
-			$this->alerts[] = array("message"=> 'Use Redis with the WP Redis object cache drop-in to speed up your backend. <a href="https://pantheon.io/docs/wordpress-redis/" target="_blank">Learn More</a>', "code" => 1);
 		} else {
-			$this->alerts[] = array("message"=> "Redis found", "code" => 0);
+			$this->alerts[] = array('message'=> 'WP Redis for object caching was found. We recommend using Object Cache Pro as a replacement. <a href="https://docs.pantheon.io/guides/object-cache-pro/installing-configuring/" target="_blank">Learn More</a> about how to install Object Cache Pro.', 'code' => 1);
+		}
+		if ( ! defined( 'WP_REDIS_CONFIG' ) || ! WP_REDIS_CONFIG ) {
```

Comment:
I think this should be an `elseif`. Otherwise, the first check will "fail" (WP Redis is installed, you should install OCP) and then the second check will _also_ fail (OCP not found, you should install OCP), resulting in two messages that are almost identical. I was in the middle of rewriting my code snippet when you merged it, I think 😅 

```suggestion
		elseif ( ! defined( 'WP_REDIS_CONFIG' ) || ! WP_REDIS_CONFIG ) {
```